src/StoreApi/Schemas/V1/CrossSellsItemSchema.php

```diff
@@ -0,0 +1,62 @@
+<?php
```

Comment:
Why are we using a custom schema instead of reusing `ProductSchema` directly?

I know you're trying to shave out some bytes here from the actual response but that's going to be negligible and there might be data on the product response that's going to be useful to third party extenders.

Reply:
Good question @senadir. That was actually the reason (reduce response complexity), but you do make a pretty good point about extensibility. Although by extending it, someone can reintroduce whatever data it needs, it does make sense to keep this object on par with WC_Product since in the end, it is a direct representation of it.

One minor tweak to your suggestion would be, instead of reusing it directly, we could still extend it into a CrossSellsSchema without change, so that in the future third-party extenders could extend ProductSchema and CrossSellsSchema separately.

Do you agree with this?

Reply:
Can we achieve this via a param passed to `get_extended_schema` without having an extra schema class? some context param maybe.

Reply:
I'm marking this as _resolved_ for now, and we can talk about it further once you come back @senadir 🙂